README.md

```diff
@@ -1 +1,156 @@
+[![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
+[![Documentation Status](https://readthedocs.org/projects/ansicolortags/badge/?version=latest)](https://docs.yugabyte.com/)
+[![Ask in forum](https://img.shields.io/badge/ask%20us-forum-orange.svg)](https://forum.yugabyte.com/)
+[![Slack chat](https://img.shields.io/badge/Slack:-%23yugabyte_db-blueviolet.svg?logo=slack)](https://www.yugabyte.com/slack)
+[![Analytics](https://yugabyte.appspot.com/UA-104956980-4/home?pixel&useReferer)](https://github.com/yugabyte/ga-beacon)
+
+
+* [What is YBBENCH Docker Container?](#what-is-yb-benchmark-docker-container)
+* [How to use](#how-to-use)
+* [Build from source code](#build-from-source-code)
+* [YBBENCH In Interactive Mode](#ybbench-in-interactive-mode)
+
+## What is YB-Bench Docker Container?
+This benchmark docker container is a way to easily run various benchmarks on the YugabyteDB.
+
+This includes following benchmarks pre-installed and ready to be used:
+1. [TPC-C](https://github.com/yugabyte/tpcc)
+2. [SYSBENCH](https://github.com/yugabyte/sysbench)
+3. [YCSB](https://github.com/yugabyte/ycsb)
+4. [YB-SAMPLE-APPS](https://github.com/yugabyte/yb-sample-apps)
+
+## How to use.
+- #### Pull the docker image
+   ```shell
+   docker pull yugabytedb/ybbench:latest
+   ```
+- #### Get the benchmark help by running the container
+   ```shell
+    docker run --name ybbench --rm -it yugabytedb/ybbench:latest
+   ```
+  Above run command will print out following help
+  ```shell
+  ******************************************************************************
+  *                                                                            *
+  *                         YugabyteDB Benchmark Help                          *
+  *                                                                            *
+  ******************************************************************************
+  *         This tool is used to run the benchmarks on YugabyteDB              *
+
+     Syntax: ./run [tpccbenchmark/sysbench/yb-sample-apps/ycsb] [params]
+
+      Benchmark Options:
+
+          tpccbenchmark                 to run tpcc benchmark.
+          sysbench                      to run sysbench benchmark
+          yb-sample-apps                to run yb-sample-apps
+          ycsb                          to run ycsb benchmark
+
+  ******************************************************************************
+  ```
+- #### Get specific benchmark help(-h/--help)
+   ```shell
+    docker run --name ybbench --rm -it yugabytedb/ybbench:latest ./run <benchmark> --help
+    # example:
+    # docker run --name ybbench --rm -it yugabytedb/ybbench:latest ./run tpccbenchmark -h
+   ```
+  
+- #### Run specific Benchmark
+  - ##### Run sample tpcc workload(Provide comma separated list of nodes). More info can be found on [TPCC](https://github.com/yugabyte/tpcc) repository.
```

Comment:
Run sample tpcc workload(Provide comma separated list of nodes). More info can be found on [TPCC](https://github.com/yugabyte/tpcc) repository.  -----> 

(title)Running TPC-C 

(In next line) Info about TPC-C specific parameters can be found [here](https://github.com/yugabyte/tpcc)


Do same for the other benchmarks